account_invoice_edifact/models/account_move.py

```diff
@@ -0,0 +1,297 @@
+                buyer.city,
+                "",
+                buyer.zip,
+                buyer.country_id.code,
+            ),
+            # Internal customer number
+            ("RFF", ["IT", buyer.id]),
+            # Buyer information
+            (
+                "NAD",
+                "BY",
+                [buyer.id, "", "92"],
+                "",
+                buyer.commercial_company_name,
+                [buyer.street, ""],
+                buyer.city,
+                "",
+                buyer.zip,
+                buyer.country_id.code,
+            ),
+            ("RFF", ["API", ""]),
+            # Seller information
+            (
+                "NAD",
+                "SE",
+                [seller_id_number.name, "", "92"],
+                "",
+                seller.commercial_company_name,
+                [seller.street, ""],
+                seller.city,
+                "",
+                seller.zip,
+                seller.country_id.code,
+            ),
+            # VAT registration number
+            ("RFF", ["VA", seller.vat]),
+            # Government reference number
+            ("RFF", ["GN", seller.vat]),  # TODO: Fix it
+            # Delivery party Information
+            (
+                "NAD",
+                "DP",
+                [shipper_id_number.name, "", "92"],
+                "",
+                shipper.commercial_company_name,
+                [shipper.street, ""],
+                shipper.city,
+                "",
+                shipper.zip,
+                shipper.country_id.code,
+            ),
+            ("RFF", ["API", ""]),
+            # Reference currency
+            ("CUX", ["2", buyer.currency_id.name, "4"]),
+            # Rate of exchange
+            ("DTM", ["134", today, "102"]),
+            ("PAT", "3"),
+            # Terms net due date
+            ("DTM", ["13", self.invoice_date_due, "102"]),
+            # Discount terms
+            (
+                "PAT",
+                "22",
+                "",
+                ["5", "3", "D", discount_days],
+            ),  # TODO: check value this again later
+            # Discount percentage
+            (
+                "PCD",
+                "12",
+                "0",
+                discount_percentage,
+            ),  # TODO: check value this again later
+            # Penalty terms
+            ("PAT", "20"),
+            # Penalty percentage
+            ("PCD", "15", "0"),  # TODO: check value this again later
+            # Allowance percentage
+            ("PCD", "1", "0", "13"),  # TODO: check value this again later
+            # Allowance or charge amount
+            ("MOA", "8", "0"),  # TODO: check value this again later
+        ]
+
+    def _edifact_export_invoice_product(self):
+        self.ensure_one()
+        number = 0
+        segments = []
+        vals = {}
+        tax = {}
+        for line in self.line_ids:
+            if line.display_type == "product":
```

Comment:
```suggestion
            if line.display_type != "product":
                continue
```
and you save a looooot of indentation in the rest of the method